configs/defaults/large_cohort.toml

```diff
@@ -1,7 +1,9 @@
 [workflow]
 name = 'large_cohort'
-scatter_count = 50
+scatter_count_genotype = 50
 status_reporter = 'metamist'
+# define reference fasta
+ref_fasta='gs://gcp-public-data--broad-references/hg38/v0/dragen_reference/Homo_sapiens_assembly38_masked.fasta'
```

Comment:
I don't mind it being overrideable, but feels awkward to constantly override it to a public source when we do have it in the bucket:

```
config['references']['broad']['ref_fasta'] = "gs://cpg-common-main/references/hg38/v0/dragen_reference/Homo_sapiens_assembly38_masked.fasta"
```

Reply:
I've given the cpg bucket path instead here 👌